File: C#/E3.AddFunctionality/S4.Refactor/CalculatorTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TDDLegacyCode.E3.AddFunctionality.S4.Refactor
{
    [TestClass]
    public class CalculatorTests : BaseCalculatorTests
    {
        [TestMethod]
        public void Number_after_minus_should_still_display()
        {
            Pressing("9-").ShouldShow("9");
        }
    }
}

File: Java/src/test/java/testingil/webinar/tddlegacycode/e3/addfunctionality/s4/refactor/CalculatorTests.java
package testingil.webinar.tddlegacycode.e3.addfunctionality.s4.refactor;

import org.junit.jupiter.api.Test;

class CalculatorTests extends BaseCalculatorTests{

	@Test
	void number_after_minus_should_still_display() {
		pressing("9-").shouldShow("9");
	}

}
